hasjob/views/index.py

```diff
@@ -90,7 +90,7 @@ def json_index(data):
     return jsonify(result)
 
 
-def fetch_jobposts(request_args, request_values, is_index, board, board_jobs, gkiosk, basequery, md5sum, domain, location, title, showall, statuses, batched, ageless, template_vars, search_query=None):
+def fetch_jobposts(request_args, request_values, is_index, board, board_jobs, gkiosk, basequery, md5sum, domain, location, title, showall, statuses, batched, ageless, template_vars, search_query=None, query_string=None):
```

Comment:
Since this is parsed and not the raw query string, shouldn't the parameter remain `search_query`?

Reply:
Okay, just noticed you're accepting (and need) both.